### PR TITLE
Add scalable vector redraw of shadps4 icon

### PR DESCRIPTION
**Pull request by fpiesche**
This is a freely-scalable source file that could also easily be used for other icon bases. The icon does not fill the entire canvas to follow [the FreeDesktop project's quality guidelines](https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/quality-guidelines/#reasonable-footprint).

This PR is a version of #1494 without the mess I made of the commit history over there...

### CODE REVIEW

**Comment by ngoquang2708** [comment]
Please add an entry to the image in the ```REUSE.toml```.

**Comment by fpiesche** [comment]
argh, I've done it again >:(

File: .github/linux-appimage-qt.sh [hunk 1]
```diff
@@ -27,7 +27,7 @@ chmod a+x linuxdeploy-plugin-checkrt-x86_64.sh
 
 cp -a "$GITHUB_WORKSPACE/build/translations" AppDir/usr/bin
 
-./linuxdeploy-x86_64.AppImage --appdir AppDir -d "$GITHUB_WORKSPACE"/.github/shadps4.desktop  -e "$GITHUB_WORKSPACE"/build/shadps4 -i "$GITHUB_WORKSPACE"/.github/shadps4.png --plugin qt 
+./linuxdeploy-x86_64.AppImage --appdir AppDir -d "$GITHUB_WORKSPACE"/.github/shadps4.desktop  -e "$GITHUB_WORKSPACE"/build/shadps4 -i "$GITHUB_WORKSPACE"/src/images/net.shadps4.shadPS4.svg --plugin qt
```
Comment (jardon) [inline .github/linux-appimage-qt.sh:30]: this will need to be updated after #1495

**Review by jardon — APPROVED** [review]
Looks good

**Comment by jardon** [comment]
actually this isnt a super noticable thing, but the right side of the "O" is a little messed up.  idk if its trivial to fix

**Comment by ngoquang2708** [comment]
> actually this isnt a super noticable thing, but the right side of the "O" is a little messed up. idk if its trivial to fix

Good catch.

**Comment by fpiesche** [comment]
It was trivial to fix, just a matter of me having set the wrong stroke style on the circle. Good spot, and it's also made me realise I still had a full copy of the .png icon in there from drawing the svg over it! That'll massively reduce the file size on this too :D

File: CMakeLists.txt [hunk 2]
```diff
@@ -941,4 +941,5 @@ install(TARGETS shadps4 BUNDLE DESTINATION .)
 if (ENABLE_QT_GUI AND CMAKE_SYSTEM_NAME STREQUAL "Linux")
     install(FILES ".github/shadps4.desktop" DESTINATION "share/applications")
     install(FILES ".github/shadps4.png" DESTINATION "share/icons/hicolor/512x512/apps")
```
Comment (abouvier) [inline CMakeLists.txt:943]: The 512x512 icon should have the same name as the scalable one.

Comment (fpiesche) [reply CMakeLists.txt:943]: https://github.com/shadps4-emu/shadPS4/pull/1495 includes a change to make that happen :)

**Pull request merged**